puppet/zulip/manifests/base.pp

```diff
@@ -58,6 +59,7 @@
     'wheezy'  => '9.1',
     'jessie'  => '9.4',
     'stretch' => '9.6',
+    'buster'  => '11',
     'precise' => '9.1',
     'trusty'  => '9.3',
     'vivid'   => '9.4',
```

Comment:
Can you split out these changes to their own commit?  Given that it's reformatting the release detection logic, it seems like that should get its own commit.

Reply:
Got it :+1: